addons/mail/static/tests/qunit_suite_tests/components/discuss_tests.js

```diff
@@ -4001,5 +4001,131 @@ QUnit.test('send message only once when enter is pressed twice quickly', async f
     );
 });
 
+QUnit.test('there should be a button to show member list in the thread view topbar initially', async function (assert) {
```

Comment:
We should create `discuss_member_list_tests` for this since we already have `discuss_sidebar_tests` and this is a 4000 LoC file.
Better split testing concern.

Reply:
Changes done.